lib/Sema/TypeCheckAttr.cpp

```diff
@@ -3954,9 +3904,14 @@ void AttributeChecker::visitTransposingAttr(TransposingAttr *attr) {
     return;
   }
 
+  bool wrtSelf = false;
+  if (!parsedWrtParams.empty())
+    wrtSelf = parsedWrtParams.front().getKind() ==
```

Comment:
Todo: make this `wrtSelf` logic more robust than checking `parsedWrtParams.front().getKind()`.